### DISCUSSION TITLE
Default to Dark Mode

### DISCUSSION THREAD

**Discussion by tzarger** [chakra-ui/ark#1157] [Q&A] [answered]
Hello, I am building a site with dark and light mode.. and I want to default to the dark mode, and not sure how to do? Anyone have an answer here?

Thanks!

**Comment by cschroeter** [comment, marked as answer]
I appreciate your question, but it's worth noting that setting a default theme (like dark mode or light mode) is generally managed at the application level, and typically not a responsibility of a headless component library.

Headless component libraries are focused on logic and state management, and intentionally avoid dictating styles or themes to ensure maximum flexibility. They should work with any kind of styling strategy you have in your application, including dark mode or light mode.
